ai-platform-unified/notebooks/unofficial/pipelines/google-cloud-pipeline-components_automl_text.ipynb

```diff
@@ -695,7 +695,9 @@
       "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        "text_classsif_pipeline.json",
</pre><p>Typo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/341/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
same as above... it was meant to be just an abbreviation of 'classification' -- if it struck you as weird maybe that's a cue to change it.